python/neuroglancer/pipeline/storage.py

```diff
@@ -225,15 +224,36 @@ def delete_file(self, file_path):
         if os.path.exists(path):
             os.remove(path)
 
-    def list_files(self, prefix):
+    def list_files(self, prefix, flat):
+        """
+        List the files in the layer with the given prefix. 
+
+        flat means only generate one level of a directory,
+        while non-flat means generate all file paths with that 
+        prefix.
+        """
+
         layer_path = self.get_path_to_file("")        
```

Comment:
`self.layer_path` is only available at the Storage level, not at the interface level :(